app/src/main/java/com/mk/tasky/agenda/detail/components/model/ReminderTypes.kt

```diff
@@ -0,0 +1,9 @@
+package com.mk.tasky.agenda.detail.components.model
+
+sealed class ReminderTypes(val type: String) {
+    object TenMinutes : ReminderTypes("10 minutes before")
```

Comment:
In case you'd like to translate your app, how would you do it for these strings?

Reply:
Added UIText to fullfill this